### DISCUSSION TITLE
Helm chart for local AI and slack bot?

### DISCUSSION THREAD

**Discussion by mohanramunc** [mudler/LocalAI#1709] [Q&A] [answered]
folks, where do I find Helm charts for k8s configuration for local AI, the slack bot specifically? where do I start? any advice would be helpful.

**Comment by mudler** [comment, marked as answer]
@mohanramunc for kubernetes you can follow these steps:

```
# Install the helm repository
helm repo add go-skynet https://go-skynet.github.io/helm-charts/
# Update the repositories
helm repo update
# Get the values
helm show values go-skynet/local-ai > values.yaml

# Edit the values value if needed
# vim values.yaml ...

# Install the helm chart
helm install local-ai go-skynet/local-ai -f values.yaml
```

You can check out the docs over here: https://localai.io/docs/getting-started/manual/

If you don't want to use helm, see also this deployment file as a starting point: https://github.com/mudler/LocalAI/blob/master/examples/kubernetes/deployment.yaml

A slack bot example is here: https://github.com/mudler/LocalAI/blob/master/examples/slack-qa-bot/README.md
Deployment file: https://github.com/mudler/LocalAI/blob/master/examples/slack-qa-bot/deployment.yaml

**Reply by mohanramunc** [reply]
Thank you!
